### DISCUSSION TITLE
Getting the tutorial experiment to work on my website

### DISCUSSION THREAD

**Discussion by legatrix** [jspsych/jsPsych#850] [Q&A] [answered]
I apologize if this is a stupid question. I just worked through the reaction time tutorial, and built the experiment, which works on my local machine. When I tried uploading the folder (containing experiment.html, /img/, and the jsPsych library) to my website, the experiment doesn't run. It does load the html page with the correct title ("My Experiment"), but otherwise just a blank screen. I have no idea what the problem is: lacking resources to javascript, insufficient permissions (don't think it's this, as I set the containing folder to 777 recursively), or something else. Any tips would be very welcome.

Thanks for your help.

**Comment by jodeleeuw** [comment, marked as answer]
Do you see any errors in the browser console when you run the experiment on your website?

**Reply by legatrix** [reply]
Yes, it failed to load the package. Through this I found the problem: In the Hello World and Reaction Time tutorials, the resources are loaded with:

`<script src = "jspsych-6.1.0/jspsych.js"></script>`

I changed this (and the other resource loading lines) to

`<script src = "jsPsych-6.1.0/jspsych.js"></script>`


(the name of the folder as downloaded from Github), and it worked. Somehow the capitalization difference doesn't cause a problem locally, but it does online. Thanks!

**Reply by jodeleeuw** [reply]
Great! Glad it is fixed.
